spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
Why remove this spec? Don't we want to make sure that there is no failed attempts limit when confirming a new phone?